### PR TITLE
Windows: Fix for CI

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

@jfrazelle @tianon @icecrime. This is a fix for Windows CI failures. It needs a change on the Windows CI servers to set DOCKER_REMOTE_DAEMON=1 where indicated below.
- export TIMEOUT=60m
- TIMEOUT=60m
- export DOCKER_HOST=tcp://10.0.0.15:2375
- DOCKER_HOST=tcp://10.0.0.15:2375
- export DOCKER_TEST_HOST=tcp://10.0.0.15:2375
- DOCKER_TEST_HOST=tcp://10.0.0.15:2375
- unset DOCKER_CLIENTONLY

<---- ADD HERE ---> export DOCKER_REMOTE_DAEMON=1
- hack/make.sh binary test-integration-cli
  # WARNING! I don't seem to be running in the Docker container.
  # The result of this command might be an incorrect build, and will not be
  #   officially supported.
  #
  # Try this instead: make all
  #

### CODE REVIEW

**Comment by jhowardmsft** [comment]
Windows CI will look horrible until the CI server is updated as indicated above.

**Comment by jfrazelle** [comment]
updating now

**Comment by jfrazelle** [comment]
updated!

**Comment by jhowardmsft** [comment]
Superstar @jfrazelle :+1:

**Comment by jhowardmsft** [comment]
Yay, Windows CI is clean again :)

**Comment by cpuguy83** [comment]
*sigh* janky...

**Comment by jhowardmsft** [comment]
All green :)

**Comment by calavera** [comment]
LGTM

File: docs/project/test-and-docs.md [hunk 1]
```diff
@@ -184,27 +185,31 @@ terminal on Windows.
...
-		$ export DOCKER_CLIENTONLY=1
-     
-	This ensures you are building only the client binary instead of both the
-	binary and the daemon.
-	
 4. Set `DOCKER_TEST_HOST` to the `tcp://IP_ADDRESS:2376` value; substitute your
-machine's actual IP address, for example:
+Linux machines actual IP address. For example:
 
 		$ export DOCKER_TEST_HOST=tcp://263.124.23.200:2376
```
Comment (thaJeztah) [inline docs/project/test-and-docs.md]: LOL, I just noticed this interesting IP-address, starting with 263..

Comment (jhowardmsft) [inline docs/project/test-and-docs.md]: Oh don't make me force an update to change it and put myself in the land of Jenkins chances... I can PR that separatel as a docs only PR :)

Comment (LK4D4) [inline docs/project/test-and-docs.md]: Haha, start opening it right now :)

**Comment by LK4D4** [comment]
LGTM

File: docs/project/test-and-docs.md (hunk 1 above)
Comment (thaJeztah) [inline docs/project/test-and-docs.md]: Haha, it's just that I must've seen that IP-address many times and only now spotted that :-)

Comment (calavera) [inline docs/project/test-and-docs.md]: my cloud IPs are in the range from 1.X.X.X to 999.X.X.X, problem? :trollface:

Comment (thaJeztah) [inline docs/project/test-and-docs.md]: @calavera are you seeing `cannot to the docker daemon....` messages?
Try with `--ip-verify=false`

**Pull request merged**